server/adaptors/integrations/__data__/repository/aws_elb/aws_elb-1.0.0.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "aws_elb",
+    "version": "1.0.0",
+    "displayName": "AWS ELB",
+    "description": "Nginx HTTP server collector",
+    "license": "Apache-2.0",
+    "type": "logs",
+    "author": "John Doe",
+    "sourceUrl": "https://github.com/",
+    "statics": {
+        "logo": {
+            "annotation": "NginX Logo",
+            "path": "logo.svg"
+        },
+        "gallery": [
+            {
+                "annotation": "NginX Dashboard",
+                "path": "dashboard1.png"
+            },
+            {
+                "annotation": "NginX Logo",
```

Comment:
Assets all need updating, Nginx -> ELB. Also, we shouldn't have a logo in the gallery (I know this was just stolen from the example integration, we should fix that later too).